dashboards-observability/public/components/visualizations/charts/logs_view/logs_view.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+import { RAW_QUERY, SELECTED_TIMESTAMP } from '../../../../../common/constants/explorer';
+import { DataGrid } from '../../../../components/event_analytics/explorer/events_views/data_grid';
+import './logs_view.scss';
+
+export const LogsView = ({ visualizations }: any) => {
+  const http = visualizations?.data?.explorer?.http;
```

Comment:
instead of checking at all places for undefined , we can have one variblae in which we can check this ondition - `isualizations?.data?.explorer?` and use that variable.